### ISSUE TITLE
PouchDB.replicate() not working for Android cordova app in release mode.

### ISSUE THREAD

**Issue by avijeetnpi** [pouchdb/pouchdb#7420]
### Issue
PouchDB.replicate() not working for Android cordova app in release mode. It works fine in debug mode. Issue is only with one way replication with  PouchDB.replicate(), PouchDB.sync() works perfectly in the same app. 

### Info
- Environment: Hybrid App(Cordova 7.1.0)
- Platform: Android
- Adapter: Default option.
- Server: CouchDB Server

### Reproduce
Following code can be used to reproduce the issue. Use same code for android(Debug), it works fine but doesn't work for android(Release).
None of the console.log() is triggered, when I try to write data to localPouchData by localPouchData.put(doc);. All of these are triggered in debug mode.

var opts = {
            live: true,
            retry: true
        };
var localPouchData= new PouchDB('mylocaldb');
var remoteCouchData = new PouchDB('https:xxx.xxx.com:6984/yourdb');

 var replication1_data = localPouchData.replicate.to(remoteCouchData, opts).on('change', function (info) {
            // handle change
            console.log("changes");
        }).on('paused', function (err) {
            // replication paused (e.g. replication up to date, user went offline)
            console.log('Offline' + err);
        }).on('active', function () {
            // replicate resumed (e.g. new changes replicating, user went back online)
            console.log('Online');
        }).on('denied', function (err) {
            // a document failed to replicate (e.g. due to permissions)
            console.log('denied' + err);
        }).on('complete', function (info) {
            // handle complete
            console.log('success' + info);
        }).on('error', function (err) {
            // handle error
            console.log('error' + err);
        });

**Comment by daleharvey** [comment]
Does release mode serve things from the filesystem as opposed to a web server? `PouchDB.sync` is a basic wrapper that does 2 `PouchDB.replicate` calls so worth keeping in mind that if sync is working, replicate is working and may be something to do with your call to replicate

Either way there is no indication that this is a pouchdb bug, cordova or all these "weblike" environments commonly have broken apis or need particular configurations to get working, if you see an indication that the bug is actually down to pouchdb please reopen, cheers

**Comment by avijeetnpi** [comment]
Hi Dale,

Thanks for replying!
Sync also stopped working after I posted this issue so I figured out something else is wrong. Actually my server certificate was changed and it magically stopped receiving requests from Android device in release mode. Same code works fine with Android in debug mode, windows 10 and iOS. So I forced my app to talk to server my adding the whole address of my server in white-list configuration of the app and it started working. It works completely fine in Android in debug mode,iOS and Windows without white-listing so may be certificate installed doesn't like android in release mode. 
Very strange issue!!
But it is working fine now 👍
